games-simulation/lincity-ng/lincity-ng-2.9_beta.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils multiprocessing
+
+DESCRIPTION="City simulation game"
+HOMEPAGE="https://github.com/lincity-ng/lincity-ng"
+SRC_URI="https://github.com/lincity-ng/lincity-ng/archive/lincity-ng-${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BitstreamVera CC-BY-SA-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="virtual/opengl
+	sys-libs/zlib
+	dev-libs/libxml2
+	media-libs/libsdl[sound,joystick,opengl,video]
+	media-libs/sdl-mixer[vorbis]
+	media-libs/sdl-image[png]
+	media-libs/sdl-ttf
+	media-libs/sdl-gfx
+	dev-games/physfs"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	dev-util/ftjam"
+
+S=${WORKDIR}/${PN}-${P/_/-}
+
+src_prepare() {
+	default
+	sed -i "/COPYING COPYING-data.txt COPYING-fonts.txt CREDITS /d" \
+		Jamfile || die
+	./autogen.sh || die
```

Comment:
I haven't seen exactly what this autogen script does but you should use `eautoreconf` from autotools eclass if that works instead.

Reply:
I tried. Unfortunately `eautoreconf` does not work here. `autogen.sh` runs aclocal, autoheader, then generates `Jamconfig.in` which is needed for `configure` and then runs autoconf.
Generating Jamconfig.in could be done in src_prepare by hand. But should it?